pkgs/os-specific/linux/usbutils/default.nix

```diff
@@ -14,6 +14,7 @@ stdenv.mkDerivation rec {
   postInstall =
     ''
       substituteInPlace $out/bin/lsusb.py \
+        --replace "/usr/bin/env python3" ${python3}/bin/python3 \
```

Comment:
You should add `python3` to `buildInputs`. `patchShebangs` will do this for you in `fixupOutputHook`.